models/gc_spider_baseline/meta.json

```diff
@@ -0,0 +1,219 @@
+          "VERTEBRAE_DISK_T6T7",
+          "VERTEBRAE_DISK_T5T6",
+          "VERTEBRAE_DISK_T4T5",
+          "VERTEBRAE_DISK_T3T4",
+          "VERTEBRAE_DISK_T2T3",
+          "VERTEBRAE_DISK_T1T2",
+          "VERTEBRAE_DISK_C7T1",
+          "VERTEBRAE_DISK_C6C7",
+          "VERTEBRAE_DISK_C5C6",
+          "VERTEBRAE_DISK_C4C5",
+          "VERTEBRAE_DISK_C3C4",
+          "VERTEBRAE_DISK_C2C3",
+          "SPINAL_CANAL"
+        ]
+      },
+      {
+        "type": "Prediction",
+        "label": "Raw Sagittal Spine Segmentation",
+        "valueType": "Raw segmentation image",
+        "description": "The raw segmentation of the segmented structures as mha. The vertebrae are subsequently labeled between 1 and 24, with 24 being the lowest lumbar vertebra (L5). Vertebra which are not completely visible within the field of view of the MR image are labeled between 101 and 124. The intervertebral discs are labeled between 201 and 224 and are decided by the vertebra above it. The spinal canal is labeled as 100."
+      }
+    ],
+    "model": {
+      "architecture": "3D U-Net-like fully-convolutional neural network",
+      "training": "supervised",
+      "cmpapproach": "2D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 447
+      },
+      "evaluation": {
+        "vol_samples": 87
+      },
+      "public": true,
+      "external": false
+    }
+  },
+  "details": {
+    "name": "SPIDER Baseline algorithm: Iterative Instance Segmentation of the spine",
+    "version": "0.1.0",
+    "devteam": "DIAGNijmegen (Diagnostic Image Analysis Group, Radboud UMC, The Netherlands)",
+    "type": "U-net",
+    "date": {
+      "weights": "",
+      "code": "2023-08-15",
+      "pub": "2023-06-23"
+    },
+    "cite": "van der Graaf, J.W., van Hooff, M.L., Buckens, C.F.M. et al. Lumbar spine segmentation in MR images: a dataset and a public benchmark. Sci Data 11, 264 (2024). doi: 10.1038/s41597-024-03090-w",
+    "license": {
+      "code": "Apache 2.0",
+      "weights": "Apache 2.0"
+    },
+    "publications": [
+      {
+        "title": "Lumbar spine segmentation in MR images: a dataset and a public benchmark",
+        "uri": "https://doi.org/10.1038/s41597-024-03090-w"
+      },
+      {
+        "title": "Iterative fully convolutional neural networks for automatic vertebra segmentation and identification",
+        "uri": "https://doi.org/10.1016/j.media.2019.02.005"
+      }
+    ],
+    "github": "https://github.com/DIAGNijmegen/SPIDER-Baseline-IIS",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "This model is intended to be used with T1 or T2 weighted MRI images of the lumbar spine in sagittal orientation only. Ensure that the lumbar spine is visible in the input MRI images to ensure reliable results. This algorithm is available on Grand Challenge [1] and is used as baseline for the SPIDER challenge [2].",
```

Comment:
> Ensure that the lumbar spine is visible in the input MRI images to ensure reliable results

The consequence (unreliable/unexpected results when the lumbar spine is absent) should also be included in the Limitations section.

The limitations section aims to give the reader fast access to information about what the model cannot.

Reply:
Added it to the limitations